src/components/cookie-banner/index.md.njk

```diff
@@ -29,95 +40,112 @@ Check with your organisation's privacy expert to see how data protection legisla
 
-{{ example({group: "components", item: "cookie-banner", example: "essential", html: true, nunjucks: true, open: false}) }}
+**[code example - cookie confirmation - server side with no additional messages]**
 
-## What to cover in your cookie banner
+#### Show the same message to all users
 
-Include the name of the service in the banner heading to help users understand that the cookies you’re talking about are different from the ones set by the main GOV.UK platform.
+In the cookie banner, tell the user about all the cookies you’re using in the service - whether they've enabled JavaScript or not. This way, you won’t need to ask the user to give their consent again on their next visit.
 
-You’ll need to change the example cookie banner text if your service:
+#### Help users keep their place using progressive enhancement
 
-- allows third parties to set cookies (tell the user that both your organisation and other organisations will be setting cookies)
-- uses cookies for reasons other than collecting analytics information or remembering the user’s settings
+If the user is entering information into a form as part of the service, they might lose their place when they submit their choice to accept or reject cookies.
 
-Keep the text as short as possible while making sure it’s an accurate description of how you use cookies. For example, if you use more than one ‘functional’ cookie and there’s not enough space to mention what each of them does, you could ask for permission to set cookies so ‘you can use as many of the service’s features as possible’.
+To help users running JavaScript on their device, you can write some JavaScript code to let them submit their choice and prevent the page from reloading.
+
+Include all possible messages that the user could see in the cookie banner when the page loads. Hide these with the `hidden` HTML attribute where needed.
 
-## When the user accepts or rejects cookies using the cookie banner
+**[code example - cookie question - server side with additional messages but hidden]**
 
-If you’re using JavaScript to manage cookie consent, write your own JavaScript so that when the user accepts or rejects cookies in the banner:
+**[code example - cookie confirmation - server side with additional messages but hidden]**
 
-- the cookie message is hidden using the `hidden` attribute
-- a replacement message is unhidden and displayed in place of the cookie banner
-- a replacement message is displayed in place of the cookie message
-- focus shifts to the replacement message
+### Option 3: If you set non-essential cookies - but only on the client
 
-Give the replacement message the `role="alert"` and `tabindex="-1"` attributes, to allow the focus to shift and so assistive technology can read the message.
+You can choose to make your banner to only work with JavaScript if your service only needs to set non-essential cookies on the client.
+ 
+Write your own JavaScript code to make your banner do what it should do.
```

Comment:
I wonder if it makes more sense to move this sentence below the next one? That way, the list of things the code needs to do directly follows this sentence. We could even combine "Write your own JavaScript code to make your banner do what it should do" and "Your code should be able to:" to avoid this slightly vague "...make your banner do what it should do".